pkgs/top-level/all-packages.nix

```diff
@@ -27737,13 +27737,12 @@ with pkgs;
 
   blktrace = callPackage ../os-specific/linux/blktrace { };
 
-  bluez5 = callPackage ../os-specific/linux/bluez { };
-
-  bluez5-experimental = callPackage ../os-specific/linux/bluez {
+  bluez-experimental = bluez.override {
     withExperimental = true;
   };
 
-  bluez = bluez5;
+  bluez5 = bluez;
+  bluez5-experimental = bluez-experimental;
```

Comment:
Shouldn't those be moved to aliases.nix and be removed from the tree?

Reply:
Maybe. I kept it because, well, it is better not to break things in a mass-rebuild.

Reply:
How should that break things? This is a noop and the treewide change can be done in an extra commit.

Reply:
By default ofborg emits an error, since it ignores aliases.nix.

Further I prefer to do this "retirement" - which includes rewriting the occurrences of `bluez5` in the Nixpkgs tree as a whole - when this modification arrives on master branch.